ostia-operator/Makefile

```diff
@@ -20,6 +20,9 @@ bash:
 push:
 	docker push $(IMAGE)
 
+unit:
```

Comment:
top command "make tests" that runs unit + integration?

Reply:
If you don't see any value in the existing `test` target (I don't particularly) then I'll replace it?